src/components/landing/EventsListItem/EventListCard/EventListCard.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from "react";
+import { NavLink } from "react-router-dom";
+import PropTypes from "prop-types";
+import PhotoService from "../../../../services/PhotoService";
+import eventDefaultImage from "../../../../constants/eventDefaultImage";
+import parseDate from "../../../helpers/parseDate";
+import IconsEventCard from "../../EventCard/IconsEventCard/IconsEventCard";
+import "./EventListCard.scss";
+
+const photoService = new PhotoService();
+const VIEW_DETAIL = "View detail";
+
+const EventListCard = ({ event }) => {
+  const { id, data, title, description, location, dateFrom } = event;
+
+  const [eventImage, setEventImage] = useState(eventDefaultImage);
+
+  useEffect(() => {
+    photoService.getFullEventPhoto(data?.id).then(eventFullImage => {
+      if (eventFullImage !== null) {
```

Comment:
Why didn't check `undefined` case?

Reply:
I don't know. It was written here before us, but I added this check ✔